pages/linux/yplan.md

```diff
@@ -0,0 +1,8 @@
+# yplan
+
+> A Perl prgram that generates LaTex code to print a vertical-type size-monthly daily planner for any chosen year.
+> More information: <https://www.ctan.org/tex-archive/macros/latex/contrib/yplan>.
+
+- Create a daily planner with specified language, lettercase (uppercase or lowercase) and year:
+
+`yplan {{language}} {{lettercase}} {{year}} > {{file.tex}}`
```

Comment:
```suggestion
`yplan {{language}} {{lettercase}} {{year}} > {{path/to/file.tex}}`
```

You're using shell redirection so it applies to non-current directory as well.

Reply:
Done